src/transformStateWithClones.js

```diff
@@ -7,7 +7,32 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const result = [];
+
+  let newState = { ...state };
+
+  for (const action of actions) {
+    switch (action.type) {
```

Comment:
3. [CODE STYLE]: switch/case should always have default case for error handling.

Need to set default case

Reply:
Add default case.

Reply:
add also destructing  `const { type, keysToRemove, extraData } = action;`

Reply:
Added destructing const { type, keysToRemove, extraData } = action;